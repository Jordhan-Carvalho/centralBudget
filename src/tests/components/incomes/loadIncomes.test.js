import React from 'react';
import { shallow } from 'enzyme';
import LoadIncomes from '../../../components/incomes/loadIncomes';

test('should render incomes page', () => {
    const wrapper = shallow(<LoadIncomes />);
    expect(wrapper).toMatchSnapshot();
});