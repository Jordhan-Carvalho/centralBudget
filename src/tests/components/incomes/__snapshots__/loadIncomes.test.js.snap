// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render incomes page 1`] = `
<Fragment>
  <h1>
    Incomes List
  </h1>
  <hr />
  <ConnectFunction />
  <ConnectFunction
    type="income"
  />
  <hr />
  <ConnectFunction />
  <hr />
  <Link
    to="/createinc"
  >
    <button
      className="btn btn-dark"
    >
      Add Income
    </button>
  </Link>
</Fragment>
`;
